audio/ffmpeg.py

```diff
@@ -112,7 +112,8 @@ def __init__(
         if self.error_threshold > 0:
             self.out_failed_files = self.output_path("failed_files.txt")
 
-        self.rqmt = {"time": 4, "cpu": 4, "mem": 8}
+        # e.g. 1 core for python and 4x2 cores for ffmpeg, one for input processing and one for output processing
```

Comment:
~Where does the 4x2 come from? Can we configure ffmpeg to use less threads? Will this number now be lower as you set `"-threads", "1"` for ffmpeg?~

Oh I see, we use `multiprocessing.pool` that is configured via the requested number of cores. But still where does the factor of 4 come from? Should we make this a parameter to the job to (more) expose it to the user?

Reply:
Yes we could expose that parameter, this is just arbitrary.

Reply:
Ok, I weakly suggest adding this as a parameter (as the new default of 9 cores might be a lot for some cluster environments.. or if you want to more effectively use the `2^x` processors of your node)

Otherwise I'm fine with the changes.